docs/GettingStartedDocs/Contributors/WindowsSGX1FLCGettingStarted.md

```diff
@@ -0,0 +1,131 @@
+# Getting Started with Open Enclave on Windows for systems with support for SGX1 with Flexible Launch Control(FLC)
+
+## Platform requirements
+
+Intel® X86-64bit architecture with SGX1 and Flexible Launch Control (FLC) support. (e.g. Intel Coffee Lake CPU)
+
+Note: To check if your system has support for SGX1 with FLC, please look [here](../SGXSupportLevel.md)
+
+A version of Windows OS with native support for SGX features:
+- For server: Windows Server 2016
+- For client: Windows 10 64-bit version 1709 or newer
+- To check your Windows version, run `winver` on the command line.
+
+## Install Git and Clone the Open Enclave SDK repo
+
+Download and install Git for Windows from [here](https://git-scm.com/download/win)
+
+Clone the Open Enclave SDK
+
+```powershell
+git clone https://github.com/openenclave/openenclave.git
+```
+
+This creates a source tree under the directory called openenclave.
+
+## Install project prerequisites
+
+First, change directory into the openenclave repository:
+
+```powershell
+cd openenclave
+```
+
+Run the following from powershell to deploy all the prerequisites for building Open Enclave (including Intel's DCAP primitives and Azure's DCAP library).
+
+```scripts/install-windows-prereqs.ps1```
+
+To install the prerequisites along with the Azure Data Center Attestation Primitives (DCAP) Client, use the below command. The Azure DCAP Client is necessary to perform attestation on an Azure Confidential Computing VM.
+
+```powershell
+cd scripts
+.\install-windows-prereqs.ps1 -InstallPath YOUR_WORKSPACE_PATH_HERE -LaunchConfiguration SGX1FLC -DCAPClientType Azure
+```
+If you would like to skip the installation of the Azure DCAP Client, use the command below.
+
+```powershell
+cd scripts
+.\install-windows-prereqs.ps1 -InstallPath YOUR_WORKSPACE_PATH_HERE -LaunchConfiguration SGX1FLC -DCAPClientType None
+```
+If there is another DCAP Client type that you would like to install, you are welcome to submit a pull request to add support for that DCAP Client.
+
+As an example, if you cloned the Open Enclave SDK repo into C:\openenclave and want to install the Azure DCAP Client, you would run the following command.
+
+```powershell
+cd scripts
+.\install-windows-prereqs.ps1 -InstallPath C:\openenclave -LaunchConfiguration SGX1FLC -DCAPClientType Azure
+```
+
+If you prefer to manually install prerequisites, please refer to this [document](WindowsManualInstallPrereqs.md).
+
+## Build
+
+Launch the [x64 Native Tools Command Prompt for VS 2017](
+https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs)
+Normally this is accessible under the `Visual Studio 2017` folder in the Start Menu.
+To build, first create a build directory ("build" in the example below) and change directory into it.
+Then run `cmake` to configure the build and generate the Makefiles, and then build by running `ninja`.
```

Comment:
Did we ever walk through installing CMake? Also, we reference GNU Makefiles here, but we don't use them, we're generating Ninja build files (which I also don't recall us walking through the installation, but maybe that's in another document?)

Reply:
+1. Both cmake and ninja are installed as part of Visual Studio, but we need to mention that they have to be explicitly selected during the installation.